WBooks/Library/Details/BookInfo/BookInfoViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  BookInfoViewController.swift
+//  WBooks
+//
+//  Created by ana.mancuso on 17/10/2022.
+//
+
+import UIKit
+
+class BookInfoViewController: UIViewController {
+    
+    //MARK: Properties
+    
+    private lazy var bookView = BookInfoView()
+    private let bookModel: BookInfoViewModel
```

Comment:
the name of the constant would be a little better if instead of Model it had ViewModel